src/Rules/Program.hs

```diff
@@ -37,8 +37,26 @@ buildProgram rs context@Context {..} = when (isProgram package) $ do
         -- We build only unwrapped binaries in inplace/lib/bin
         inplaceLibBinPath -/- programName context <.> exe %> \bin -> do
             binStage <- installStage
+            buildBinaryAndWrapperLib rs (context { stage = binStage }) bin
+
+        inplaceLibBinPath -/- programName context <.> "bin" <.> exe %> \bin -> do
```

Comment:
It sounds a bit far-fetched, because there are no wrappers at all on Windows at the moment. It's unclear whether they will ever be needed, even if `iserv` is added someday. If this does happen, then adding a few `exe` in the right spots will be easy, but right now it looks like a potential source of confusion.